csrmetrics/csrmetrics.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package csrmetrics contains metrics definitions to be recorded for monitoring purposes
+package csrmetrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// OutboundRPCStatus is a status string of the outbound RPC metric contains either status code or status string
+type OutboundRPCStatus string
+
+// Status constants for metrics
+const (
+	OutboundRPCStatusNotFound OutboundRPCStatus = "not_found"
```

Comment:
unused?

Reply:
yes, two of them are currently unused but can be used in the future.